INSTALL-vagrant.md

```diff
@@ -49,11 +49,17 @@ vagrant up
 vagrant ssh
 ```
 
-5) once in via SSH, you can now start the automated installer:
+5) once in via SSH, you can now start the automated installer.
```

Comment:
This is a minor nitpick, but if someone is reading this tutorial they are pretty likely to run the command you give them _before_ seeing your alternative below. Maybe just add "optional_event_name" to the end of the `./setup_vagrant_control_server.sh` command.